text/0000-expand-math.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: Expand `math`
+- Start Date: 2022-02-28
+- RFC PR:
+- Pony Issue:
+
+# Summary
+
+This RFC proposes expanding the standard library's `math` package and the shape it might take to leverage the full extent of Pony. There was prior discussion on [Zulip](https://ponylang.zulipchat.com/#narrow/stream/192795-contribute-to.20Pony/topic/math.20lib) years ago, but I expect some opinions to have changed.
+
+# Motivation
+
+Currently, `math` includes limited functionality. This should be expanded to include math types, constants, and present a structure with further expansion in mind. An expanded `math` library will allow a unified mathematics among Pony developers.
+
+# Detailed design
+
+The primary goals of this initial expansion are: 
+
+1. restructure the `math` library into distinct sub-packages
+2. provide common `math` data types
```

Comment:
what does this mean? It feels important to understanding this RFC, but I don't know what it means.

Reply:
This was my attempt to state that there are two parts to this RFC. The first is to "expand" math by the creation of subpackages to organize its functionality, while the second is to "expand" math by filling these subpackages with new functionality.

Reply:
what is a common math data type?

Reply:
I added examples to this line to address your question. I was using "math" and "numeric" interchangeably here and meant to imply what is listed below in the RFC, namely `Rational`, `BigInt`, and `Complex` types which are common enough in usage to be part of the standard library.